massgov/mayflower
# DP-10282: Replacing DangerouslySetInnerHTML with the React Parser

## Description
Changed
Minor
(React) DP-10242: Replacing DangerouslySetInnerHTML with the React Parser in the Paragraph, Unordered List, Ordered List and Contact Group components.

## Related Issue / Ticket
- [JIRA issue](https://jira.mass.gov/browse/DP-10282)

## Steps to Test
1. Check out branch
1. `nvm use && npm install`
1. `npm run start`
1. Navigate to atom->text->Paragraph.
1. Observer text appears properly.
1. Navigate to atom->lists->OrderedLists
1. Observe text appears properly.
1. Navigate to atom->lists->UnorderedLists
1. Observe text appears properly.

#### @TODO


1. Navigate to molecules->ContactGroup
1. TODO: Why won't the address appear in this?
